diwako_cbrn.vr/scripts/cbrn/functions/fn_loadoutEH.sqf

```diff
@@ -31,10 +36,13 @@ if (!(_unit getVariable ["cbrn_mask_on", false]) && {_hasMask}) then {
 if (_unit getVariable ["cbrn_mask_on", false] && {!_hasMask}) then {
     // guy JUST put that mask away
     _unit setVariable ["cbrn_mask_on", false, true];
+	_unit setVariable ["cbrn_mask_fogging", false];
+	_unit setVariable ["cbrn_mask_fogged", false];
```

Comment:
Do those also need handling on death or respawn?

Reply:
It (probably) did, has been added for both cases in [8d801c0](https://github.com/diwako/diwako_cbrn/pull/5/commits/8d801c0645c71189a829282407c9f619139e93d7)